### ISSUE TITLE
Calibre-web connection?

### ISSUE THREAD

**Issue by psyciknz** [shermp/Kobo-UNCaGED#49]
I have both calibre on a desktop and calibre-web (via docker).
After initially not being able to connect to calibre-web, I started the content server on calibre.....and then the wireless service.  It now seems to connect.

Is there an equivalent on calibre-web for this wireless service?  what does it do?

**Comment by NiLuJe** [comment]
No, calibre-web is an OPDS front-end to a calibre database, like calibre's own content server.

The wireless service is a backend for clients (such as UNCaGED) to act as real calibre device drivers. That sorta imply direct interaction with the actual calibre GUI.

**Comment by psyciknz** [comment]
thanks, I think i just worked that out in calibre, and saw that you can end to a device when it's connected.   Rather than use the device to browse a library and pull via opds.
